Dojo.Generators.Core/Utils/FileSystemUtils.cs

```diff
@@ -28,5 +31,16 @@ internal static string[] FindFilesWithExtension(string folder, string extension)
 
             return files;
         }
+
+        internal static IEnumerable<string> FindFilesWithExtensions(string folder, params string[] extensions)
+        {
+            // Since on .netstandard2.0 EnumerationOptions cannot be used
+            var files = extensions.SelectMany(extension => Directory.GetFiles(
```

Comment:
nit: you iterate over directory multiple times (disk seeks). Consider instead do it once for all files, and filter file name in your code

Reply:
this code was just moved, but sadly there is not C# method that allows you to search by multiple extensions at once.
the only option would be to retrive all files, and filter in memory afterwards, but that might end up being more expensive as we do not know how many files could there be in the respective directory.